cfn/encoding/unmarshal.go

```diff
@@ -13,6 +13,9 @@ func Unmarshal(data []byte, v interface{}) error {
 	}
 
 	Unstringify(dataMap, v)
```

Comment:
Should this be `err = Unstringify(dataMap, v)` ?

Reply:
Yes, that was a typo.